src/include/autoinstall/autoinst_dialogs.rb

```diff
@@ -82,28 +81,24 @@ def ProfileSourceDialog(original)
     # @return [String] device
     def DiskSelectionDialog
       Builtins.y2milestone("Selecting disk manually....")
-      tm = Storage.GetTargetMap
+      devicegraph = Y2Storage::StorageManager.instance.y2storage_probed
+      disks = devicegraph.disk_devices
       contents = Dummy()
 
-      if Ops.greater_than(Builtins.size(tm), 0)
+      if !disks.empty?
         buttonbox = VBox()
 
         i = 0
-        Builtins.foreach(tm) do |tname, tdata|
-          if Storage.IsRealDisk(tdata)
-            tlinename = Ops.get_string(tdata, "name", "?")
-            tline = Ops.add(
-              Ops.add(Ops.add("&", Ops.add(i, 1)), ":    "),
-              tlinename
-            )
-            sel = Storage.GetPartDisk == tname &&
-              Storage.GetPartMode != "CUSTOM"
-            buttonbox = Builtins.add(
-              buttonbox,
-              Left(RadioButton(Id(tname), tline, sel))
-            )
-            i = Ops.add(i, 1)
-          end
+        disks.each do |disk|
```

Comment:
Yest, it should be each_win_index, IMO too. But since we didn't fix it during the original code review, I will leave it as is (supposedly manually tested) for the time being.